prow/images/buildpack-golang-toolbox/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Basic golang buildpack
+
+FROM eu.gcr.io/kyma-project/prow/test-infra/bootstrap:v20181204-a6e79be
```

Comment:
Why don't you use a golang buildpack image as the base? This way you can remove the repetitive parts and only keep the additional tools part